<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>655</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>637</width>
    <height>911</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>COMODATAR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagens/logo.ico</normaloff>imagens/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 181, 156);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbPesquisa_cliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>cod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cnpj</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edtPesquisa_cliente">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridCliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>621</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CNPJ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Endereço</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telefone</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnPesquisar_maquina">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Imagens/Lupa.png</normaloff>../../../../Imagens/Lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pesquisa de cliente:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridMaquina">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>621</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Patrimonio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descriçao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Situaçao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Numero de serie</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Voltagem</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnPesquisar_cliente">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Imagens/Lupa.png</normaloff>../../../../Imagens/Lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPesquisa_maquina">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPesquisa_maquina">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>numero_de_serie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>patrimonio</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pesquisa de comodado:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>610</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Comodatar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>610</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
